### ISSUE TITLE
gosec docker image has been removed, which breaks github actions

### ISSUE THREAD

**Issue by asalzmann** [cosmos/gosec#32]
### Summary 

informalsystems/gosec@master docker image was removed

### Steps to reproduce the behavior

run github action, github action fails

examples of failed runs
evmos: https://github.com/evmos/evmos/runs/7704031692?check_suite_focus=true
stride: https://github.com/Stride-Labs/stride/runs/7714955110?check_suite_focus=true

### Expected behavior

github action runs

### Actual behavior

github action cannot pull docker image

```
Error: Docker pull failed with exit code 1
```

**Comment by odeke-em** [comment]
Oops, sorry about this, we have been working on making it a GitHub action. Kindly tagging @kirbyquerby

**Comment by asalzmann** [comment]
I think it was previously a working github action (we had it configured and running until a few days ago)

**Comment by kirbyquerby** [comment]
There wasn't ever actually a docker image for informalsystems/gosec -- the action previously pointed to securego/gosec but was changed recently in #31 : https://github.com/informalsystems/gosec/commit/2c09f8898cc7e6dfedac771097f965e7b78cbf64#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6L13

Thus, everyone who has been using the informalsystems/gosec action has been running regular gosec this whole time. Our change to action.yml still isn't correct, though, since the plan is to upload the image to ghcr rather than docker hub.

@odeke-em Until we can get a successful image upload, the fix should be to revert the change to action.yml. Then, once the image is working, we can switch it over.

**Comment by odeke-em** [comment]
Thank you @kirbyquerby! I've reverted that commit and we can use our fork to work on this and this reach out to the Informal Systems folks to give write permissions for the Docker image. @asalzmann please try again.

**Comment by fedekunze** [comment]
here's the full error message:

```
/usr/bin/docker pull informalsystems/gosec
  Using default tag: latest
  Error response from daemon: pull access denied for informalsystems/gosec, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

**Comment by odeke-em** [comment]
> here's the full error message:
> 
> 
> 
> ```
> 
> /usr/bin/docker pull informalsystems/gosec
> 
>   Using default tag: latest
> 
>   Error response from daemon: pull access denied for informalsystems/gosec, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
> 
> ```

@fedekunze yeah that's an entirely new error that happened since Friday as we still need the Informal Systems team to set write permissions to allow Docker images to be deployed. I reverted the commits since Friday and it should go back to working as it was before. Please try again.

**Comment by asalzmann** [comment]
We switched over to the following action for now
```

name: Gosec
on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]
jobs:
  tests:
    runs-on: ubuntu-latest
    env:
      GO111MODULE: on
    steps:
      - name: Checkout Source
        uses: actions/checkout@v3
      - name: Run Gosec Security Scanner
        uses: securego/gosec@master
        with:
          args: -exclude-dir=deps -severity=high ./...
```

It might make sense to remove the action instructions entirely until the docker image is up, given they're misleading (they imply the informalsystems/gosec docker image exists and that you can set up informalsystems gosec to run as an action)

**Comment by odeke-em** [comment]
@asalzmann we finally got the ability to build a Docker image and it is now available as ghcr.io/cosmos/gosec:latest, please update to that image, then once satisfied, please close this issue. Thank you!

**Comment by kirbyquerby** [comment]
The image is uploaded but this issue is related to the action, which isn't ready yet.

**Comment by kirbyquerby** [comment]
@asalzmann This should be fully fixed -- we now have a [docker image](https://github.com/cosmos/gosec/pkgs/container/gosec) available and the action was set up in #37.

Feel free to reopen if something's still not working.

**Comment by asalzmann** [comment]
Nice! Will give this a go
